src/frontend/src/pages/ChangeRequestsPage/ChangeRequestsTable.tsx

```diff
@@ -120,12 +121,22 @@ const ChangeRequestsTable: React.FC = () => {
         title={'Change Requests'}
```

Comment:
Can you add a little space between the pageTitle and the table? You should be able to wrap the PageTitle in a div and use the style prop on that like the sx property on the mui button to set MarginBottom to like 10-20 or smthn, whatever looks best. Lmk if that doesn't make sense